wp-content/themes/haruoshi_origin/functions.php

```diff
@@ -106,3 +106,50 @@ function codex_custom_init()
 add_action('init', 'codex_custom_init');
 
 add_theme_support('post-thumbnails');
+
+if( function_exists('acf_add_local_field_group') ):
+acf_add_local_field_group(array(
+	'key' => 'group_62261c6bd16b5',
+	'title' => 'Works',
+	'fields' => array(
+		array(
+			'key' => 'field_62404882234c7',
```

Comment:
```suggestion
	'key' => '<ここ>',
	'title' => 'Works',
	'fields' => array(
		array(
			'key' => '<ここ>',
```
「ここ」って書いたところはACFが勝手に生成したkeyだから、key指定でああだこうだしたい場合は自分がわかりやすいkeyに変更しておくといいらしい。
おれはできるだけ変更するようにしてるけど今まで役に立ったことはないから、修正しなくてもいい気がする

Reply:
done https://github.com/oooshima/origin_wordpress/pull/33/commits/e04b68cf043084d6aba64529340d16b69cfe2f4c